### PR TITLE
[13576] Alert component

### PR DESCRIPTION
**Pull request by ibrkhalil**
image
image
image
image

fixes #13625

### Summary

Alert component for quo2, just like the RN one.

#### Platforms
- Android

status: ready

### CODE REVIEW

**Review by flexsurfer — COMMENTED** [review]
seems like Podfile changes not related to this PR and could be reverted

**Comment by flexsurfer** [comment]
seems like we have alert implemented https://github.com/status-im/status-react/blob/develop/src/status_im/utils/utils.cljs

File: src/quo/react_native.cljs [hunk 1]
```diff
@@ -55,6 +55,14 @@
 
 (def keyboard (.-Keyboard ^js rn))
 
+(defn alert [alert-title alert-description btns]
```
Comment (flexsurfer) [inline src/quo/react_native.cljs:58]: if we move to quo2, probably we should move react native as well, but honestly im not sure if it should be in quo at all , we need to discuss

**Comment by ibrkhalil** [comment]
> seems like we have alert implemented https://github.com/status-im/status-react/blob/develop/src/status_im/utils/utils.cljs

Oh, I have not seen that
I think then the alert component does not need anymore work, right ?

**Comment by flexsurfer** [comment]
i guess we could move it in quo2

**Comment by flexsurfer** [comment]
anyway this one can be closed

**Pull request closed without merging**